src/Entities/Line.php

```diff
@@ -44,7 +44,7 @@ public function __construct(
         string            $merchantOrderLineId,
         string            $name,
         int               $quantity,
-        float             $amount,
+        float              $amount,
         int               $vatPercentage,
         private ?Currency $currency = null,
```

Comment:
Looks like here is an extra space
 ` float              $amount,`

Reply:
In my IDE there is no additionall space, idk, maybe bug